gap/Magic.gi

```diff
@@ -293,6 +293,13 @@ function( arg )
         gapdoc := rec();
     fi;
 
+    #
+    # Allow for legacy gapdoc_latex_options
+    #
+    if IsBound( scaffold.gapdoc_latex_options ) and not IsBound( gapdoc.GAPDocLaTeXOptions ) then 
+        gapdoc.GAPDocLaTeXOptions := scaffold.gapdoc_latex_options; 
```

Comment:
I would suggest to also issue a warning here, say `Info(InfoWarning, 1, "Please replace the DEPRECATED option <scaffold.gapdoc_latex_options> by <gapdoc.GapDocLaTeXOptions>");`

Reply:
Perhaps we also call it just `gapdoc.LaTeXOptions` instead? The extra `GapDoc` seems redundant? Perhaps @sebasguts also has an opinion on this?